x/erc20/types/proposal.go

```diff
@@ -112,3 +117,19 @@ func (ttcp *ToggleTokenConversionProposal) ValidateBasic() error {
 
 	return v1beta1.ValidateAbstract(ttcp)
 }
+
+// ProposalRoute returns router key for this proposal.
+// RegisterCoinProposal is DEPRECATED remove after v16 upgrade
+func (*RegisterCoinProposal) ProposalRoute() string { return RouterKey }
+
+// ProposalType returns proposal type for this proposal.
+// RegisterCoinProposal is DEPRECATED remove after v16 upgrade
+func (*RegisterCoinProposal) ProposalType() string {
+	return ProposalTypeRegisterCoin
+}
+
+// ValidateBasic performs a stateless check of the proposal fields.
+// RegisterCoinProposal is DEPRECATED remove after v16 upgrade
+func (rtbp *RegisterCoinProposal) ValidateBasic() error {
+	return errors.New("Deprecated")
```

Comment:
nit: I think usually errors should lower case in SDK applications
```suggestion
	return errors.New("deprecated")
```